cmake/Modules/FindMKL.cmake

```diff
@@ -73,8 +73,8 @@ if(USE_MKL_IDEEP_OR_MKLML)
           set(MKLDNN_LIB "${CMAKE_SHARED_LIBRARY_PREFIX}mkldnn${CMAKE_SHARED_LIBRARY_SUFFIX}")
           list(APPEND IDEEP_LIBRARIES "${PROJECT_BINARY_DIR}/lib/${MKLDNN_LIB}")
           set(CAFFE2_USE_IDEEP 1)
-          # Do NOT use MPI if IDEEP is enabled
-          set(USE_MPI OFF)
+          # Do NOT use OpenMP if IDEEP is enabled
+          set(USE_OPENMP OFF)
```

Comment:
Why do we turn this off?

Reply:
Originally, used to avoid confilit of two omp libs between current system and mkldnn internal.
But sounds like it's not needed for now on. Will remove it.